account_financial_risk/wizards/account_invoice_state_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account_invoice_confirm_view" model="ir.ui.view">
+        <field name="name">account.invoice.confirm.form</field>
+        <field name="model">account.invoice.confirm</field>
+        <field name="inherit_id" ref="account.account_invoice_confirm_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//footer" position="before">
+                <p attrs="{'invisible': [('info_risk','=', '')]}">
```

Comment:
Does this work with empty fields (instead of `('info_risk', '=', False)`)?

Reply:
https://github.com/OCA/credit-control/pull/17/files#diff-9a2973bb509867614553981c26e976ecR15

Reply:
`('info_risk', '=', False)` don't work